packages/components/pie-assistive-text/src/index.ts

```diff
@@ -25,17 +25,6 @@ export class PieAssistiveText extends LitElement implements AssistiveTextProps {
     @validPropertyValues(componentSelector, variants, 'default')
     public variant?: AssistiveTextProps['variant'] = 'default';
 
-    /**
- * Renders the assistive-text icon content.
- * @private
- */
-    private renderIcon (): TemplateResult {
-        const { variant } = this;
-        return html`
-            ${variant === 'success' ? html`<icon-check-circle class="c-assistiveText-icon" size="s" />` : nothing}
```

Comment:
I've just noticed we're using self-closing tags for the icons in the previous code. This is prone to causing rendering errors. Please could you test if simply adding closing tags to the icon elements fixes the issue @siggerzz ?